cartography/intel/aws/ec2/images.py

```diff
@@ -17,9 +16,9 @@
 @timeit
 @aws_handle_regions
 def get_images(boto3_session: boto3.session.Session, region: str) -> List[Dict]:
-    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
```

Comment:
per the other PR, let's leave all the configs in here, and do that as a separate PR

Reply:
Will do.